prombench/Makefile

```diff
@@ -11,37 +11,87 @@ deploy: nodepool_create resource_apply
 clean: resource_delete nodepool_delete
 
 nodepool_create:
-	$(INFRA_CMD) gke nodepool create -a ${AUTH_FILE} \
-		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
-		-f manifests/prombench/nodepools.yaml
+	ifeq (${PROVIDER}, gke)
+		${INFRA_CMD} ${PROVIDER} nodepool create -a ${AUTH_FILE} \
+			-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
+			-f manifests/prombench/nodes_${PROVIDER}.yaml
+	endif
+	ifeq ($(PROVIDER), eks)
```

Comment:
I think we don't need these `ifeq` blocks, you can just include all the deployment vars and deployment range vars in here. The ones that won't be specified will be empty and since the manifest files for that provider will not be using those empty deployment vars, this is a non-issue.

What do you think?